Makefile

```diff
@@ -1,18 +1,22 @@
 CACHITO_COMPOSE_ENGINE ?= docker-compose
+PYTHON_VERSION_VENV ?= python3.8
+PYTHON_VERSION_TOX ?= py38
+PYTHON_TEST_SUITE ?=
 ifeq ($(CACHITO_COMPOSE_ENGINE), docker-compose)
 	DOWN_OPTS=-v
 endif
 
-all: run
+all: venv run-start
 
-clean:
-	$(CACHITO_COMPOSE_ENGINE) down $(DOWN_OPTS)
+clean: run-down
 	rm -rf venv && rm -rf *.egg-info && rm -rf dist && rm -rf *.log* && rm -rf .tox && rm -rf tmp
 
+.PHONY: venv
 venv:
-	virtualenv --python=python3 venv && venv/bin/python setup.py develop && venv/bin/pip install -r requirements.txt -r requirements-web.txt tox
+	virtualenv --python=${PYTHON_VERSION_VENV} --system-site-packages venv && venv/bin/python setup.py develop && venv/bin/pip install --upgrade pip && venv/bin/pip install -r requirements.txt -r requirements-web.txt tox
```

Comment:
Why do we need `--system-site-packages`?

Reply:
I cannot remember the specific error since I did this so long ago, but if I remember, I was hitting an error installing the packages without this parameter. Adding `--system-site-packages` was the only resolution that I could find at the time... Trying to reproduce to see.

Reply:
I cannot reproduce this now, so I will remove it since it seems _better_ to have full isolation.